src/game.cpp

```diff
@@ -2593,12 +2593,12 @@ void Game::playerRequestTrade(uint32_t playerId, const Position& pos, uint8_t st
 
 	Player* tradePartner = getPlayerByID(tradePlayerId);
 	if (!tradePartner || tradePartner == player) {
-		player->sendTextMessage(MESSAGE_INFO_DESCR, "Sorry, not possible.");
```

Comment:
Why this change? It will now only show on the bottom of the screen as white text, is it correct behaviour?

Reply:
Yes. All messages except for "x wants to trade with you" come as MESSAGE_STATUS_SMALL